tools/test_data_to_json/test_data_to_json.py

```diff
@@ -0,0 +1,116 @@
+from optparse import OptionParser
```

Comment:
Please add a comment to the top of this file with a license block (ideally BSD if you're willing)